<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>482</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>48</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>9999999</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2" rowspan="2">
     <widget class="QPushButton" name="okButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QPushButton" name="addButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="2">
     <widget class="QPushButton" name="cancelButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="numofqueLine">
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>999999</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Number of Questions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>411</width>
     <height>82</height>
    </rect>
   </property>
   <property name="title">
    <string>Type</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QRadioButton" name="multiplechoiceButton">
      <property name="text">
       <string>Multiple Choice</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="shortanswerButton">
      <property name="text">
       <string>Short Answer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="difficultyBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>59</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Difficulty</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="4">
     <widget class="QPushButton" name="hardButton">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Hard</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="mediumButton">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="easyButton">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Easy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>341</width>
     <height>471</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="2" column="0">
     <widget class="QGraphicsView" name="questionView">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="reloadButton">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
